src/app/collections/PresentationNode.scss

```diff
@@ -25,7 +25,9 @@
       }
     }
 
-    #triumphs & {
+    #triumphs &,
+    #legacyTriumphs &,
+    #collections & {
```

Comment:
Can we just do `.presentation-node` or whatever the selector is?

Reply:
if you want sure. it was originally this way just to provide control because i wasn't sure what pnodes might be used for in the future

Reply:
Either way is fine - I was just a bit surprised to see these called out by ID.